cmd/kured/main.go

```diff
@@ -122,6 +124,11 @@ func rebootRequired() bool {
 func rebootBlocked() bool {
 	if prometheusURL != "" {
 		alertNames, err := alerts.PrometheusActiveAlerts(prometheusURL, alertFilter)
+		weekday := time.Now().Weekday()
+		if noWeekendReboot && (int(weekday) == 0 || int(weekday) == 6) {
```

Comment:
suggest `time.Sunday` and `time.Saturday`

Reply:
Thanks for the relevant comment, I have updated the PR according to the comment by  @tgarlot  and skipped the magic numbers as part of that commit